### DISCUSSION TITLE
Outdated item on the "before merge" checklist?

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#5842] [Work In Progress - GUI] [answered]
We have an item on the PR template checklist:
```
If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run ide build` and `./run ide watch`.
```
Given that `./run ide watch` also runs electron, is this still needed?

@wdanilo as you have power over GH templates.

**Comment by wdanilo** [comment, marked as answer]
@farmaazon `./run ide watch` uses Electron that is not packaged, it hacks into Electron and runs it with our content. There MIGHT be cases where the `./run ide build` will fail while `./run ide watch` will not. However, I do not see why the opposite could happen. I mean, if `./run ide build` succeeds and the electron app works, `./run ide watch` should as well. So I'd remove the later. What about that?

**Reply by farmaazon** [reply]
Seems reasonable.

**Reply by wdanilo** [reply]
Done: https://github.com/enso-org/enso/blob/develop/.github/PULL_REQUEST_TEMPLATE.md
